src/pheromones/hivemind-host/include/hivemind-host/UserCallDestinationDTO.h

```diff
@@ -0,0 +1,23 @@
+#ifndef __USERCALLDESTINATION_H_
+#define __USERCALLDESTINATION_H_
+
+#include <user-call.pb.h>
+
+/**@brief The destination of a user call*/
```

Comment:
We might want to explain with more details the different IDs that we need to pass. In `Message`, there is a `sourceId` and a `destinationId`, while in `UserCall`, there are also some ids to pass. Perhaps explain the distinction between the two somewhere in a docstring?

Reply:
I've added a bit of doc, i'm not sure if it's enough, how would you formulate it?

Reply:
Yeah that looks good. The rest will probably need to be explained in more details in the wiki. I just wanted to clarify the fact that there are two types of IDs: one for the distant robot+hiveBoard compound (`sourceId` and `destinationId`) and one for the element inside the compound (the IDs in the `UserCall`). Hence the fact that we have a `Message` that addresses a remote compound, and that message contains a `UserCall` that further addresses a module within the compound (either the host robot or Buzz).